### ISSUE TITLE
Is this still maintained?

### ISSUE THREAD

**Issue by deeplook** [heremaps/pptk#56]

**Comment by jiapei-nexera** [comment]
@deeplook 
Same question here...
**pptk** seems to be a very very good tool...

**Comment by SuaveFool** [comment]
Just check the last modified date, 4 years ago now, so unfortunately it doesn't seem so. Also, getting a working build of this is at least not easy given the dependencies are so old now.

**Comment by deeplook** [comment]
> Just check the last modified date, 4 years ago now, so unfortunately it doesn't seem so. Also, getting a working build of this is at least not easy given the dependencies are so old now.

Yep, it's a very sad but good example of companies with a culture and mindset unfit for the open source world. They couldn't exist without it, but are unwilling to contribute in a really meaningful way.

**Comment by nikste** [comment]
Hi, I'm taking some time to revive this project, since it has been stale some time now, and i actually really liked it.                                                                                                                        

You can install it with:
`pip install pptk-revived` 

It's a drop-in replacement — just import pptk works as before. I'm working through the open issues here one by one. Contributions and feedback are welcome!
